.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Merge with `copier-ci.yml`?

Reply:
I removed copier-ci in favour of this file